website/server/controllers/top-level/dataexport.js

```diff
@@ -97,7 +112,26 @@ async function _getUserDataForExport (user) {
  * @apiGroup DataExport
  * @apiDescription NOTE: Part of the private API that may change at any time.
  *
- * @apiSuccess {String} A json file
```

Comment:
It should probably stat that it's a json file of the user object and link to the get user route.